lib/engine/g_1856/corporation.rb

```diff
@@ -16,6 +17,7 @@ def initialize(game, sym:, name:, **opts)
         @started = false
         @escrow = nil
         super(sym: sym, name: name, **opts)
+        @all_shares = shares_by_corporation[self].dup
         @capitalization = nil
       end
 
```

Comment:
The floating behavior in 1856 is going to keep the 1856::Corporation class around although escrow logic can probably be pulled into the base classes at some point which could reduce the amount of custom 1856 code

Reply:
what about it?

Reply:
You've mentioned a desire to put the custom logic in `G1856::Coporation` into the 1856 game file before. I have some ideas on how to incrementally remove some of the custom classes for 1856 but I think that the floating behavior is going to keep the custom 1856 corporation around for that purpose